src/js/media-chrome-listbox.js

```diff
@@ -7,6 +7,28 @@ template.innerHTML = `
 <style>
   :host ul {
     list-style: none;
+    display: flex;
+    flex-direction: column;
+    gap: 0.5em;
+    margin: 0;
+    padding: 0.5em;
+    /* background-color: var(--media-control-background, rgba(20,20,30, 0.7)); */
+    background-color: black;
+    color: #ffffff;
```

Comment:
is this color overridable if a user sets the CSS via `media-captions-menu-button`?

Reply:
I added a CSS variable. Otherwise, the default color outside media-controller won't work with the default background color.